practice/00-HTML-CSS-basics/First exercise.html

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
+<html>
+<head>My Todo list</head>
+<body>
+hola mundo
```

Comment:
Don't add content alone in the body without a tag. Wrap it on a p tag or something similar.